src/Helpers/esForms.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * File that holds all public helpers to be used in the other projects.
+ *
+ * @package EightshiftForms\Helpers
+ */
+
+use EightshiftForms\Settings\Settings\SettingsGeneral;
+use EightshiftForms\Settings\SettingsHelper;
+
+/**
+ * Outputs the forms custom unique name set in the settings by provided form ID.
+ *
+ * @example echo esFormsGetFormIdByName('22826');
+ *
+ * @param string $formId Form ID.
+ *
+ * @return string
+ */
+function esFormsGetFormIdByName(string $formId): string
```

Comment:
Where will this be used? Why do we need to define a function in the global namespace?

Reply:
in your theme or other plugin